### PR TITLE
Preview update on extensions (Seo / Excerpt) changes

### PR DESCRIPTION
**Pull request by Prokyonn**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Fixed tickets | fixes https://github.com/sulu/sulu/issues/7348
| License | MIT

#### What's in this PR?

Adds updates to the page preview when seo/excerpt fields are changed

#### Why?

https://github.com/sulu/sulu/issues/7348

### CODE REVIEW

File: src/Sulu/Bundle/PageBundle/Preview/PageObjectProvider.php [hunk 1]
```diff
@@ -73,6 +74,11 @@ public function setValues($object, $locale, array $data)
         $structure = $object->getStructure();
         foreach ($data as $property => $value) {
             try {
+                if ($property === 'ext') {
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/PageBundle/Preview/PageObjectProvider.php]:
```suggestion
                if ('ext' === $property) {
```

yoda-giphy

**Comment by Prokyonn** [comment]
@alexander-schranz @wachterjohannes should we merge this to 2.4 or 2.6?

**Comment by wachterjohannes** [comment]
@Prokyonn would say 2.4

**Comment by Olivier127** [comment]
@alexander-schranz @Prokyonn  May be do the same fix in ArticleObjectProvider.php in the ArticleBundle ?

**Comment by alexander-schranz** [comment]
@Olivier127 thx for the hint do you want to create a pull request for it?

**Comment by Olivier127** [comment]
@alexander-schranz I have tried, but i'm not allowed to create any branch on ArticleBundle.

**Comment by mamazu** [comment]
You have to fork the repository and create a pull request from your repository to the repository in the sulu namespace.

**Comment by Olivier127** [comment]
@mamazu thanks. My PR : https://github.com/sulu/SuluArticleBundle/pull/676

**Pull request merged**